src/test/java/SubwayStationTest.java

```diff
@@ -0,0 +1,43 @@
+import com.google.gson.Gson;
+import org.junit.Test;
+
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class SubwayStationTest {
+
+
+
+    @Test
+    public void getName() {
+        SubwayStations stations = new SubwayStations();
+
+        try {
+            // create Gson instance
+            Gson gson = new Gson();
+
+            // create a reader
+            Reader reader = Files.newBufferedReader(Paths.get("/Users/ahuvacheifetz/Downloads/SubwayStationApplication/src/main/resources/stations.json"));
```

Comment:
Do not use a path that will only work on your machine.

Reply:
I couldn't get that to work